python/mlc_chat/nn/kv_cache.py

```diff
@@ -477,10 +479,20 @@ def _attention_prefill(h_kv, h_q, d, dtype, target: Target):  # pylint: disable=
     group_size = h_q // h_kv
     sm_scale = 1.0 / math.sqrt(float(d)) * math.log2(math.exp(1))
 
+    bdx = 32
```

Comment:
I'm not exactly sure about substituting `32` with `bdx` in the entire `_attention_prefill()` and `_attention_prefill_ragged()`; is it the intended semantics?